### PR TITLE
GitHub docs

### PR DESCRIPTION
**Pull request by alicezha0**
Updated some of the wording in the Git and Github pages; added updated photos where necessary

### CODE REVIEW

File: docs/source/git-and-github/github-basics.rst [hunk 1]
```diff
@@ -18,17 +18,13 @@ Creating a Repository
 Creating a repository is simple. We'll do this through the `GitHub Website <https://www.github.com>`_. First go to the `AguaClara Team Page <https://github.com/AguaClara>`_.
 
 
-.. image:: https://github.com/AguaClara/aguaclara_tutorial/wiki/Images/AguaClaraHome.png
-   :target: https://github.com/AguaClara/aguaclara_tutorial/wiki/Images/AguaClaraHome.png
-   :alt: AguaClaraHomePage
+.. image:: images/git11.PNG
```
Comment (HannahSi) [inline docs/source/git-and-github/github-basics.rst]: The preview on Github isn't showing this image, I think it needs to be `images/git11.png` (lowercase).

**Review by HannahSi — COMMENTED** [review]
This looks really good! Since we're no longer teaching Atom though, could you move the "Github for Atom" section to a new file and include it in the table of contents (in index.html) under "Archived"?

**Comment by alicezha0** [comment]
#57

**Comment by HannahSi** [comment]
Thanks for the updates! Some of the new images in Git in the Command Line aren't showing up again because their actual file names don't match the file name in the RST file (these are case sensitive). You can see which images aren't showing up by clicking on the RST file in Github.

**Review by HannahSi — APPROVED** [review]
This looks good to me now!

**Pull request merged**